packages/flutter/lib/src/cupertino/scrollbar.dart

```diff
@@ -125,6 +126,54 @@ class CupertinoScrollbar extends StatefulWidget {
   /// {@endtemplate}
   final ScrollController controller;
 
+  /// {@template flutter.cupertino.cupertinoScrollbar.displayAlways}
+  /// A [bool] property for showing [Scrollbar] always.
+  ///
+  /// To show [ScrollBar] always you need to make this property true,
+  /// however the [controller] shouldn't be null.
+  ///
+  /// You should pass the [Scrollable]'s [ScrollController] to [controller].
+  ///
+  /// {@tool snippet}
+  ///
```

Comment:
What are you trying to illustrate with the two different scrollbars below?  Maybe add some text here about what's going on.

Reply:
Also, what do you think about making this a live dartpad example?  You can just change the `@tool` tag above like [here](https://github.com/flutter/flutter/blob/d444582f3e152b34127d2b4dcae8bd2500c2a96e/packages/flutter/lib/src/material/app_bar.dart#L92) as an example, if you think it would be beneficial.

Reply:
> What are you trying to illustrate with the two different scrollbars below? Maybe add some text here about what's going on.

Since we are using the doc template so We are using the same doc comments for both the Material and the Cupertino styles, I just want to show basic examples for both widget `CupertinoScrollbar` and `Scrollbar` too.

Reply:
Oh I see, similar to the `controller` docs.  No need to change it, I think this makes sense.